yb-voyager/cmd/constants.go

```diff
@@ -46,9 +46,11 @@ const (
 	TARGET_DB                     = "target"
 	FF_DB                         = "ff"
 	FF_DB_IMPORTER_ROLE           = "ff_db_importer"
+	FB_DB_IMPORTER_ROLE           = "fb_db_importer"
 	TARGET_DB_IMPORTER_ROLE       = "target_db_importer"
 	SOURCE_DB_EXPORTER_ROLE       = "source_db_exporter"
-	TARGET_DB_EXPORTER_ROLE       = "target_db_exporter"
+	TARGET_DB_EXPORTER_FF_ROLE    = "target_db_exporter_ff"
+	TARGET_DB_EXPORTER_FB_ROLE    = "target_db_exporter_fb"
```

Comment:
Why do we need to distinguish between these two roles?

Reply:
I created a new exporter role here, because the trigger which the exporter deals with is different. Depending on whether we are in fall-forward/fall-back, the exporter needs to monitor for and subsequently create different triggers

Lmk if you think this can be dealt with in a better way.. It felt simpler to me to have the trigger mapped to the exporterRole 1:1